src/sentry/auth/providers/oauth2.py

```diff
@@ -125,6 +125,7 @@ def dispatch(self, request: Request, helper) -> Response:
 class OAuth2Provider(Provider, abc.ABC):
     client_id = None
     client_secret = None
+    is_partner = False
```

Comment:
Does this need to be here if `is_partner = False` is already defined in Provider?

Reply:
This was a stacked PR :P I screwed up rebase. Let me fix those